rust/headless-client/src/lib.rs

```diff
@@ -274,9 +273,17 @@ impl Callbacks for CallbackHandler {
             .expect("should be able to tell the main thread that we disconnected");
     }
 
-    fn on_update_resources(&self, resources: Vec<connlib_client_shared::ResourceDescription>) {
+    fn on_update_resources(
+        &self,
+        resources: Vec<connlib_client_shared::callbacks::ResourceDescription>,
+    ) {
         // See easily with `export RUST_LOG=firezone_headless_client=debug`
-        tracing::debug!(len = resources.len(), "Printing the resource list one time");
+
+        let status = resources
+            .iter()
+            .map(|r| (r.name(), r.status()))
+            .collect::<HashMap<&str, Status>>();
+        tracing::info!(?status, "Resource Status");
```

Comment:
Same question about logging resources to INFO.

Reply:
ah, this was mostly for debugging, I'll just remove it